### ISSUE TITLE
request: visual indicator while in insert mode

### ISSUE THREAD

**Issue by collinpeters** [cmcaine/tridactyl#155]
In cVim when you enter insert mode, a hint/indicator is shown in the bottom right. This is tremendously helpful when toggling in an out of insert mode, otherwise you often don't know which mode you are in.

![selection_001](https://user-images.githubusercontent.com/1203246/33347398-95729fda-d447-11e7-9693-c9b687347c96.jpg)

**Comment by johnbeard** [comment]
Related to #57, which also suffers from lack of feedback in that indicator.

In Vimperator, it's a little "chip" in the bottom left, in the status bar (status bar mentioned in #90)

**Comment by bovine3dom** [comment]
Closed as duplicate of #57, blocked by status bar / feedback to users in #90. Thanks for the feedback!

**Comment by Soundtoxin** [comment]
Is it possible to get something more large/clear/colorful like what qutebrowser uses? I find it jarring going from qutebrowser to LibreWolf (when a site isn't working in qb, usually because of cloudflare).
